RELEASE_NOTES.md

```diff
@@ -6,10 +6,20 @@ documentation improvements. This page lists some of the highlights.
 ## New
 
 ### New Software Support
-CASTS resolved
-CAST-32797 SMA rsyslog-collector pod file descriptor leak and resource exhaustion
-CAST-32143  CSM 1.3 unbound is losing connection to upstream DNS
 
+* Resolved CASTS:
+    * `CAST-31486`: `cray-powerdns-manager` not adding compute nodes to external DNS
+    * `CAST-31712`: Unbound slamming DNS server with requests
```

Comment:
This was also fixed in CSM 1.4.2, so it's not exactly a new fix in CSM 1.5.0. If we're listing everything that was fixed in the previous CSM patch releases, this list could be a lot longer. :)